build/ansible/roles/victorialogs/files/Dockerfile.victorialogs

```diff
@@ -0,0 +1,16 @@
+# TODO: the image should be percona/pmm-server:3 once PMM v3 is released.
+
+# To build the image, run the following in the project root directory (mind the dot!):
+# docker buildx build --platform=linux/amd64 --progress=plain -t perconalab/pmm-server:victorialogs-0.37.0 -f ./build/ansible/roles/victorialogs/files/Dockerfile.victorialogs .
+FROM perconalab/pmm-server:3-dev-latest
+
+ENV GF_ANALYTICS_CHECK_FOR_UPDATES=false
+ENV GF_ANALYTICS_REPORTING_ENABLED=false
+
+USER root
+
+COPY build/ansible /opt/ansible
+
+RUN ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm-docker/victorialogs.yml
+
+USER pmm
```

Comment:
any reason to create separate docker image instead of integrating to existing one?

Reply:
seems like it leads to complication of current logic by creating duplicated files

Reply:
Yes, the reason is that anybody can build it on their own machine, which will take under one minute. No need to mess with our complicated build system.

Once we make up our minds as to which logging solution we want to go ahead with, it won't be difficult to convert it to a regular feature build.